builder/bundle_control.go

```diff
@@ -341,16 +346,11 @@ func (b *Builder) getFullMixBundleSet() (bundleSet, error) {
 	if err != nil {
 		return nil, err
 	}
-	// Add the included and optional included bundles to the mix
-	for _, bundle := range set {
-		err := b.AddBundles(bundle.DirectIncludes, false, false, false)
-		if err != nil {
-			return nil, err
-		}
-		err = b.AddBundles(bundle.OptionalIncludes, false, false, false)
-		if err != nil {
-			return nil, err
-		}
+
+	// Write final mix bundle list back to file
+	err = b.writeMixBundleList(set)
```

Comment:
The set of bundles is already recursively resolved with includes and also-adds, so it is safe to directly write the result to disk.

With this implementation, bundles added to the mix by dependencies will be added silently without prints to stdout. I think the print dump to stdout can be overwhelming and hard to read as the number of bundles drastically increases. Maybe it makes more sense to improve Mixer's ability to log and write this information to a log file.